gateway/api/server.go

```diff
@@ -276,10 +276,14 @@ func (api *Api) buildRoutes(r *apiroutes.Router) {
 		apiroutes.ReadOnlyAccessRole,
 		r.AuthMiddleware,
 		apiconnections.ListDatabases)
```

Comment:
Does this endpoint is still used by the frontend after merging these changes?

Reply:
Yes, we still list databases for Postgres and MongoDB